decidim-core/lib/decidim/organization_settings.rb

```diff
@@ -106,7 +106,8 @@ def defaults_hash
             "allowed_file_extensions" => {
               "default" => %w(jpg jpeg gif png bmp pdf rtf txt),
               "admin" => %w(jpg jpeg gif png bmp pdf doc docx xls xlsx ppt pptx ppx rtf txt odt ott odf otg ods ots),
-              "image" => %w(jpg jpeg gif png bmp ico)
+              "image" => %w(jpg jpeg gif png bmp ico),
```

Comment:
```suggestion
              "image" => %w(jpg jpeg gif png bmp),
```

As it's asked by the AC: 

> - [ ] Given that I'm an implementer
When I generate a new application
and I go to /system
Then I see that there isn't the ICO extension anymore

So unless there's a good reason for leave it, I said that we can drop `ico` file extension from here